src/Microsoft.AspNet.Routing/Logging/RouteValuesLogValues.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Framework.Logging;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNet.Routing.Logging
+{
+    public class RouteValuesLogValues : ILogValues
```

Comment:
I get why you're doing this, but it's a deviation from the pattern. Let's get buy-in to this being necessary and good.